docs/odh-rhoai/connect-vscode-to-rhoai-wb.md

```diff
@@ -0,0 +1,61 @@
+# Connect to RHOAI Workbench Kernel from local VS Code
+
+Some customers have expressed their desire to work directly on their local IDE and execute the Jupyter notebook(s) using the kernel on remote workbench running on RHOAI. Most IDEs provide connection to a remote Kernel as a standard feature. However, this standard feature does not work with RHOAI because of the way authentication to workbench is set up in RHOAI<br/>
```

Comment:
Could you rephrase to remove "customers"? All the recipes here can also be used on upstream ODH, so we try to keep this agnostic. Mentioning RHOAI is ok though.